contracts/weapons.sol

```diff
@@ -236,9 +236,15 @@ contract Weapons is Initializable, ERC721Upgradeable, AccessControlUpgradeable {
         return tokenID;
     }
 
-    function getRandomProperties(uint256 stars, uint256 seed) public pure returns (uint16) {
+    function getRandomProperties(uint256 stars, uint256 seed, uint256 choosenElement) public pure returns (uint16) {
+        uint256 trait;
+        if (choosenElement == 4) {
```

Comment:
"chosen" also we really should not use 4 as the sentinel value for this; it should be higher, like 100. We don't plan to add more elements, but if we did, we don't want to have to redo this as well.